bookmarks/queries.py

```diff
@@ -59,6 +62,24 @@ def _base_bookmarks_query(user: Optional[User], query_string: str) -> QuerySet:
             unread=True
         )
 
+    # Ensure dates are converted to correct `datetime` representation
+    dates = parse_query_dates(query_dates)
+
+    # Filter by dates
+    date_filters = {
+        'since_added': 'date_added__gte',
+        'until_added': 'date_added__lte',
+        'since_modified': 'date_modified__gte',
+        'until_modified': 'date_modified__lte'
```

Comment:
Have you thought about making one boundary exclusive? It's probably not a big deal in practice, but maybe you had some thoughts.